rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -12,22 +12,37 @@
     context:
       - Python
       - Container
+      - Pipeline
     ztrigger:
       - pr:
           CRON: ""
-          STAGES: "Allocate Resources, Connect Slave, Build {context} Artifacts, Cleanup"
+          STAGES: "Allocate Resources, Connect Slave, Prepare Deployment, Build {context} Artifacts, Cleanup"
       - periodic:
           branches: "do_not_build_on_pr"
-          STAGES: "Allocate Resources, Connect Slave, Build Python Artifacts, Build Container Artifacts, Cleanup"
+          STAGES: "Allocate Resources, Connect Slave, Prepare Deployment, Build Python Artifacts, Build Container Artifacts, Cleanup"
           PUSH_TO_MIRROR: yes
+    exclude:
+      - context: Python
+        ztrigger: periodic
+      - context: Container
+        ztrigger: periodic
+      - context: Pipeline
+        ztrigger: pr
     jobs:
       - 'RPC-Artifact_Build-{series}-{image}-{context}-{ztrigger}'
 
 - job-template:
     # DEFAULTS
     IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    DEPLOY_SWIFT: "yes"
+    DEPLOY_CEPH: "yes"
+    DEPLOY_ELK: "yes"
     CRON: "H H(9-21) * * 1-5"
-    STAGES: "Allocate Resources, Connect Slave, Build Python Artifacts, Build Container Artifacts, Cleanup"
+    CONTEXT_USER_VARS: ""
```

Comment:
I thought that they were required as they're included by default in rpc_params with no global defaults?